src/olympia/abuse/management/commands/sync_cinder_policies.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia.abuse.tasks import sync_cinder_policies
+
+
+class Command(BaseCommand):
+    help = (
+        'Sync all cinder policies to our local database. Syntax: \n'
+        '    ./manage.py sync_cinder_policies'
```

Comment:
Usage syntax is already automatically included by Django when help is printed, you don't need to add it to the help string.

Reply:
TIL. Again, copied from existing command.